### PR TITLE
bower-based asset management and some basic styling for all-posts page

### PR DESCRIPTION
**Pull request by jywarren**
This is untested, as I'm still trying to get the system installed (https://github.com/zenhack/iron-blogger2/issues/16), but could work, maybe?

### CODE REVIEW

**Comment by zenhack** [comment]
Thanks for doing this! A couple things:

* I'm getting 404's everywhere. (I realize you weren't able to test this). You should probably be using `url_for` in the templates; see http://flask.pocoo.org/docs/0.10/quickstart/#static-files
* Because of the changes I pushed to the README, there are some merge conflicts now; please fix.
* Given that the README recommends using virtualenv to do the python stuff locally, it probably makes more sense to do the same with node (i.e. no `-g` flag). Could you update the instructions (and `.gitignore`) accordingly? I also need to investigate what needs to happen on openshift to make bower work, I'll get back to you on that.

**Comment by zenhack** [comment]
Getting to do the openshift bits doesn't look too bad.

**Comment by jywarren** [comment]
I rebased this with url_for additions, but haven't yet tested it (still trying to get my virtualenv setup going) but will soon!

**Comment by jywarren** [comment]
All good now. Ready to merge!

File: README.md [hunk 1]
```diff
@@ -37,6 +37,13 @@ be started with:
 At which point, the application will be available at
 <http://localhost:5000>.
 
+###Static assets
```
Comment (zenhack) [inline README.md]: I think it probably makes more sense to put this above the bit about launching the server --- fits more with how you'd logically want to go about setting things up. Also I don't see any reason why this should be a third level heading as opposed to a second level? Change that third `#` to a space.

File: README.md [hunk 2]
```diff
@@ -37,6 +37,13 @@ be started with:
...
+
+Static assets are managed via Bower (http://bower.io) and can be installed with the command `bower install`. This helps keep our standard css/js/font libraries orderly. 
+
+Bower itself can be installed with `npm install bower`
```
Comment (zenhack) [inline README.md]: Might make sense to add `node_modules` to `.gitignore`?

**Comment by zenhack** [comment]
This looks gorgeous! thanks for doing this. I made a few minor comments inline, if you could address those.

File: README.md (hunk 1 above)
Comment (jywarren) [inline README.md]: done!

**Comment by jywarren** [comment]
Fixed! Thanks, I always read the IB planet page and not the RSS feed, so I have a stake in making it look nice! :-)

**Comment by zenhack** [comment]
Great! Merging!

**Pull request merged**